src/domains/mapDomain.ml

```diff
@@ -263,7 +263,7 @@ module HConsed (M: S) : S with
   type key = M.key and
   type value = M.value =
 struct
-  include Lattice.HConsed (M)
+  include Lattice.HConsed (M) (struct let assume_idempotent = false end)
```

Comment:
Is this used by the BaseAnalysis? If it is , the this counters the effect of having the short-circuiting, as int values will often live in these.

Reply:
It's not used at all, base uses `HashCached`.